example/testapp/apitest/features/step_definitions/steps.js

```diff
@@ -3,7 +3,11 @@ var exec = require('sync-exec')
 var cucumberApi = require('../../../../../index.js');
 
 resetDatabase = function() {
-  exec("../src/serv --reset-db")
+  exec("cp ../src/db/empty.db ../src/db/test.db")
```

Comment:
Ideally this shouldn't be a step but rather should be in the `support/feature_hooks.js` within the `BeforeFeatures` hook.  Then the `truncateTables` function should be in the main `Before` hook within `support/scenario_hooks.js`.